modules/making-open-source-more-inclusive.adoc

```diff
@@ -0,0 +1,6 @@
+[preface]
+
+[id="making-open-source-more-inclusive_{context}"]
+= Making open source more inclusive
+
+Red Hat is committed to replacing problematic language in our code, documentation, and web properties. We are beginning with these four terms: master, slave, blacklist, and whitelist. Because of the enormity of this endeavor, these changes will be implemented gradually over several upcoming releases. For more details, see link:https://www.redhat.com/en/blog/making-open-source-more-inclusive-eradicating-problematic-language[our CTO Chris Wright's message].
```

Comment:
```suggestion
Red Hat is committed to replacing problematic language in our code, documentation, and web properties. 
We are beginning with these four terms: master, slave, blacklist, and whitelist. 
Because of the enormity of this endeavor, these changes will be implemented gradually over several upcoming releases. 
For more details, see link:https://www.redhat.com/en/blog/making-open-source-more-inclusive-eradicating-problematic-language[our CTO Chris Wright's message].
```